javascript/javascript1/week2/StudentManager.js

```diff
@@ -0,0 +1,57 @@
+//Studen Manager
+
+
+const class07Students = [];
+function addStudentToClass(studentName) {
+  // You write code here
+  // studentName ==' '|| studentName == null
+  if( typeof (studentName) !== "string" ){
+    return console.log('Invalid name .. please enter valid name')
+
+  }else if(class07Students.length >=6 && studentName !=='Queen'){
+    return console.log("Cannot add more students to class 07.")
+
+}else {
+  for (let i = 0; i < class07Students.length; i++) {
+    if (studentName === class07Students[i]){
+      return console.log('Stuednt '+studentName+' is already in the class')
```

Comment:
I think you want to return a string and not the result of the console.log(...)
you could something like this  console.log(addStudentToClass(studentName))
```suggestion
      return 'Student'+studentName+' is already in the class'
```

Reply:
i tried to do that but it dose not working , it did not implement the statement  when i re add the same name it giving me : 
 console.log('Student '+studentName +' is added to the class') it is the last else in function .

Reply:
const response = 'Student'+studentName+' is already in the class';
return response,;

Reply:
![Screenshot (3)](https://user-images.githubusercontent.com/75618135/106737122-9892c000-6616-11eb-9f8d-97c99b16e8b0.png)
![Screenshot (4)](https://user-images.githubusercontent.com/75618135/106737215-b3fdcb00-6616-11eb-9b2d-6c92da8eaed2.png)

Thank for helping , but i still don´t understand what do you mean at :you could do something like this console.log(addStudentToClass(studentName))

you mean like i did here or the same order inside if  statement 
Thanks in Advance

Reply:
so i think now  can i merge PR ?